src/catalogue/items/catalogueItemsDialog.component.tsx

```diff
@@ -605,6 +605,37 @@ function CatalogueItemsDialog(props: CatalogueItemsDialogProps) {
     [errorMessages, propertyErrors]
   );
 
+
+  const options = ():Array<Manufacturer & {isrecent: string}> => {
+    const classifiedManufacturers = manufacturerList ? (manufacturerList).map((option) => {
+      return {
+        ...option,
+        isrecent: "A-Z"
+      };
+    }) : [];
+
+    //duplicating the recent manufacturers, so that they appear twice.
+    const date_now = (new Date()).getTime()
+    const recentManufacturers = classifiedManufacturers
+    .filter((option) => {
+      const created_date = (new Date(option.created_time)).getTime()
+      //dates are in ms; numerical expression is equivalent to 10 minutes
+      let isRecent = date_now - 1000*60*10 <= created_date
```

Comment:
There are some semi colons missing in this function. 
Can you change this number into a variable please `1000*60*10`


https://github.com/ral-facilities/inventory-management-system/pull/892#discussion_r1740607006
https://github.com/ral-facilities/inventory-management-system/pull/892#discussion_r1743507149

Reply:
Cheers for pointing it out, added them here and in other files now.
Ah right, sorry misunderstood you in the earlier comment.